README.md

```diff
@@ -1,63 +1,200 @@
 
-![dancing](https://media.giphy.com/media/u5dIIXcW2I1gs/giphy.gif)
+![UITableViewController](https://s3.amazonaws.com/learn-verified/uitableviewcontroller-xcode.png)
 
-* Present the problem with which `UINavigationController` solves before jumping right into creating one!
-* When then showing them how to create a `UINavigationController`, do so as if you're explaining it to a five year old, include screenshots.
-* The method I want to show them in creating a `UINavigationController` is as follows:
-* Select the `UITableViewController` in Interface Builder
+Once you've created the table view controller, don't forget to set it as the _initial view controller_. Select **Table View Controller Scene** in Interface Builder's object listing, then bring up the Attributes Inspector and check the box marked **Is Initial View Controller**. A grey arrow pointing to the table view should appear in Interface Builder.
 
-![selectedVS](http://i.imgur.com/FgHDU6W.png)
+![Is Initial View Controller](https://s3.amazonaws.com/learn-verified/attributes-initial-view-controller.png)
 
-* Editor --> Embed In --> Navigation Controller
+Finally, you should set the table view controller's class to be `TableViewController`, which is the subclass in which you will be writing your own table view controller code. Select **Table View Controller** from the object listing, bring up the Identity Inspector, and set the class to `TableViewController`.
 
-![Nav](http://i.imgur.com/CrXYt3s.png)
+![Identity Inspector](https://s3.amazonaws.com/learn-verified/identity-inspector-class.png)
 
-* I changed title of nav bar in Interface Builder (which is something we should show the student), should we also show them how to do this in `viewDidLoad()`? I'm willing to discuss this, it's important for them to know this, maybe this is thrown into a future lesson.
+One more thing to do for the time being. Remember the whole system table view controllers do to allocate and dequeue cells? You need to set the _reuse identifier_ for your table view's cells. You will then use this reuse identifier to dequeue the appropriate cell from the table view controller. Select **Table View Cell** in the object listing, then bring up the Attributes Inspector. See where it says **Reuse Identifier** near the top? Type "DirtyCell" into that box and hit <kbd>Return</kdb>. That's the identifier you'll use to dequeue your table view's cells.
 
+![Reuse Identifier](https://s3.amazonaws.com/learn-verified/attributes-inspector-reuse-identifier.png)
+
+Great! You're done...for now. You can build and run your app if you want to, although it won't do anything particularly useful yet. You're about to fill in the useful bits of the app.
+
```

Comment:
I think adding it to the prior reading makes sense. I thought I had at least discussed dequeueing in the previous lesson.